<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdeminstracionProducto</class>
 <widget class="QWidget" name="AdeminstracionProducto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="4">
    <widget class="QTreeWidget" name="ProductosTreeWidget">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">4</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">5</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">6</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="NuevoPushButton">
     <property name="text">
      <string>Nuevo</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="ActualizarPushButton">
     <property name="text">
      <string>Actualizar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="EliminarPushButton">
     <property name="text">
      <string>Eliminar </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
